.web.yml

```diff
@@ -0,0 +1,25 @@
+tutorials:
```

Comment:
is there a way to make sure there's an entry for each notebook?

Reply:
do we want that? example: no clear way to render drybell right now

Reply:
there's just the one master webpage config here, so easy to add once if we notice something is missing for website

Reply:
if we're not requiring every tutorial to be rendered on the website, this is okay